pyaerocom/data/emep_variables.ini

```diff
@@ -68,7 +68,8 @@ concrdn=SURF_ugN_RDN
 concso2=SURF_ug_SO2
 concso4=SURF_ug_SO4
 concss=SURF_ug_SS
-concsspm25=SURF_ug_SEASALT_F
+#concsspm25=SURF_ug_SEASALT_F
+concssf=SURF_ug_SEASALT_F
```

Comment:
I guess this is still concsspm25 in ebas_config.ini. Also you probably need to add the new variable concssf in variables.ini.